### PR TITLE
Allow rails_best_practices gem to use config file at /config dir

### PR DESCRIPTION
**Pull request by vladimirbazhanov**
When we exclude 'config' dir from check only default config will be used

### CODE REVIEW

**Comment by timurvafin** [comment]
@vladimirbazhanov thanks, looks OK. Could you please update CHANGE log?

**Comment by vladimirbazhanov** [comment]
done

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Introduce [CSSComb](https://github.com/csscomb/csscomb.js). For more details see: https://github.com/fs/rails-base/pull/284
 - Add `Brewfile` to track application dependencies.
+- Fixed rails-best-practices configuration file using
```
Comment (timurvafin) [inline CHANGELOG.md]: @vladimirbazhanov could you please move this entry to the top and change it a little bit:
> Fix `rails_best_practices` invocation from `bin/quality`, `config/rails_best_practices.yml` was ignored before.

**Comment by timurvafin** [comment]
@vladimirbazhanov could you please apply last comment and sqash commits before merging?

**Comment by vladimirbazhanov** [comment]
done with markdown filenames. Any other comments?)

**Comment by timurvafin** [comment]
@vladimirbazhanov :ok:

**Pull request merged**